import_package.md

```diff
@@ -0,0 +1,281 @@
+# Import package
+
+An import package is a .zip file containing a **manifest.yaml** that describes additional 
```

Comment:
The first sentence suggests that it "just" contains manifest.yaml. I think we should slightly rephrase this, suggesting the user that it contains manifest.yaml and additional files, and then clarify in the next sentence that those additional files *should* be referenced in the manifest.yaml.